docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '3'
+
+services:
+    demo_lti1p3_nginx:
+        container_name: demo_lti1p3_nginx
+        image: nginx:stable
+        networks:
+            - demo_lti1p3_network
+        ports:
+            - "8888:80"
+        expose:
+          - 443
+        volumes:
+            - .:/var/www/html:cached
+            - ./docker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf:cached
+        working_dir: /etc/nginx/conf.d
+
+    demo_lti1p3_phpfpm:
+        container_name: demo_lti1p3_phpfpm
+        build:
+            context: ./docker/phpfpm
+        expose:
+            - 9000
+        networks:
+            - demo_lti1p3_network
+        volumes:
+            - .:/var/www/html:cached
+        working_dir: /var/www/html
+
+    demo_lti1p3_redis:
+        container_name: demo_lti1p3_redis
+        image: redis:latest
+        command: ["redis-server", "--appendonly", "yes"]
+        hostname: demo_lti1p3_redis
+        networks:
+            - demo_lti1p3_network
+        ports:
+            - "6379:6379"
+        volumes:
+            - demo_lti1p3_redis_volume:/data
+
+    demo_lti1p3_redis_commander:
```

Comment:
Please put this container into `demo_lti1p3_network`, otherwise `redis-commander` won't see the `redis` container.

Reply:
it actually does, but yeah better be explicit